packages/core/src/components/hotkeys/hotkey.tsx

```diff
@@ -58,10 +86,16 @@ export class Hotkey extends AbstractComponent<IHotkeyProps, {}> {
     }
 
     public render() {
-        const { combo, label } = this.props;
+        const { allowInInput, combo, disabled, label, preventDefault, stopPropagation } = this.props;
         return <div className="pt-hotkey">
             <div className="pt-hotkey-label">{label}</div>
-            <KeyCombo combo={combo} />
+            <KeyCombo
+                allowInInput={allowInInput}
+                combo={combo}
+                disabled={disabled}
+                preventDefault={preventDefault}
+                stopPropagation={stopPropagation}
```

Comment:
seems like a reasonable candidate for a spread (due to sheer number of direct copies), but explicitness is nice too 🤷‍♂️